app/javascript/react/containers/App.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import 'babel-polyfill';
+import BeersContainer from './BeersContainer';
+import { Router, Route, IndexRoute, browserHistory } from 'react-router';
+const App = (props) => {
+  return (
+    <Router history={browserHistory}>
+      <Route path='/' component={BeersContainer}>
```

Comment:
Please make routes and divs self-closing when appropriate, to improve readability:
```suggestion
      <Route path='/' component={BeersContainer} />
```